xbmc/video/windows/GUIWindowVideoNav.cpp

```diff
@@ -268,6 +268,112 @@ std::string CGUIWindowVideoNav::GetQuickpathName(const std::string& strPath) con
   }
 }
 
+int CGUIWindowVideoNav::SelectFirstUnwatchedItem()
```

Comment:
Where should the enum be declared? Is it ok to declare it locally to CGUIWindowVideoNav?